syncstorage/storage/sql/dbconnect.py

```diff
@@ -112,19 +112,33 @@ def __new__(cls, *args, **kwds):
 PAYLOAD_TYPE = PAYLOAD_TYPE.with_variant(postgresql.TEXT(), 'postgresql')
 
 
-def _get_bso_columns(table_name):
+# Common column definitions between BSO and batch upload item tables
+
+def _common_columns(ttl):
     return (
-        Column("userid", Integer, primary_key=True, nullable=False,
-               autoincrement=False),
-        Column("collection", Integer, primary_key=True, nullable=False,
-               autoincrement=False),
-        Column("id", String(64), primary_key=True, autoincrement=False),
         Column("sortindex", Integer),
         Column("modified", BigInteger, nullable=False),
+        # MySQL no likey "DEFAULT ''".  Results in an error:
+        #     ERROR 1101 (42000): BLOB, TEXT, GEOMETRY or JSON column 'payload'
+        #                         can't have a default value
+        # Column("payload", PAYLOAD_TYPE, nullable=False),
```

Comment:
Ouch, not sure why we haven't hit this before :-/.  I guess we may as well fix it as part of this work, unless it turns out to be a rabbit-hole.